### PR TITLE
Document `--release` flag in the kubectl-gs `template nodepool` command

### PR DESCRIPTION
**Pull request by axbarsan**

### CODE REVIEW

File: src/content/ui-api/kubectl-gs/template-cluster.md [hunk 1]
```diff
@@ -54,8 +54,8 @@ It supports the following flags:
 - `--provider` - The infrastructure provider (either `aws` or `azure`).
 - `--name` - Unique identifier of the cluster.
 - `--organization` - Name of the organization that will "own" the cluster.
-- `--release` - valid workload cluster release version.
-  Can be retrieved with `gsctl list releases` for your installation. Only versions above *10.x.x*+ support cluster CRs.
+- `--release` - The Giant Swarm cluster release version.
+  Can be retrieved with `kubectl get releases` for your installation. Only versions above *10.x.x*+ support cluster CRs.
```
Comment (anvddriesch) [inline src/content/ui-api/kubectl-gs/template-cluster.md:58]: I think we can drop the part about v10.x.x now :)

File: src/content/ui-api/kubectl-gs/template-cluster.md [hunk 2]
```diff
@@ -54,11 +54,11 @@ It supports the following flags:
 - `--provider` - The infrastructure provider (either `aws` or `azure`).
 - `--name` - Unique identifier of the cluster.
 - `--organization` - Name of the organization that will "own" the cluster.
-- `--release` - valid workload cluster release version.
-  Can be retrieved with `gsctl list releases` for your installation. Only versions above *10.x.x*+ support cluster CRs.
+- `--release` - The Giant Swarm cluster release version.
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/template-cluster.md:57]: It's really called "workload cluster release" in most places currently

File: src/content/ui-api/kubectl-gs/template-nodepool.md [hunk 3]
```diff
@@ -55,6 +55,7 @@ Here are the supported flags:
 - `--nodes-min` - minimum number of worker nodes for the node pool. (default 3)
 - `--output` - Sets a file path to write the output to. If not set, standard output will be used.
 - `--organization` - Name of the organization owning the cluster.
+- `--release` - The parent workload cluster's Giant Swarm release version.
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/template-nodepool.md:58]: "Parent" isn't needed. A node pool belongs to a workload cluster.

"Giant Swarm release" is not the term we use.

```suggestion
- `--release` - The workload cluster release version.
```

**Review by anvddriesch — APPROVED** [review]

**Review by marians — APPROVED** [review]
LGTM!

Could you please update the title to mention both commands affected? Thx!

**Pull request merged**